### DISCUSSION TITLE
Port forwarding should work for wireguard?

### DISCUSSION THREAD

**Discussion by ntn888** [tenseiken/docker-qbittorrent-wireguard#10] [General]
HI, should the port forwarding work with wireguard setup? In my case it's blocked inbound (the status bar says firewalled)..

**Comment by tenseiken** [comment]
If you're using one of the paid Proton VPN servers that supports port forwarding, yes, it works.

**Reply by ntn888** [reply]
Thank you very much for the reply. I'm self-hosting the wireguard server on my remote VPS (using the standard docker template here https://github.com/linuxserver/docker-wireguard#docker-compose-recommended-click-here-for-more-info). So in this case there's no port forwarding?

**Comment by tenseiken** [comment]
I'm not sure I follow. This container manages the Wireguard connection process. I don't see why you would need to link it up with another instance of Wireguard running in another container.

Regardless, unless the endpoint of your VPN is one of the aforementioned Proton VPN servers, the automatic port forwarding in this container probably isn't going to work.

**Reply by ntn888** [reply]
oh okay..

the other instance is the server as I'm self-hosting it.. I'm not using a provider.

Thanks again for your attention.

**Comment by tenseiken** [comment]
Oh, yeah, I don't think it'd work like that.

**Comment by ntn888** [comment]
okay cheers!
